### PR TITLE
Add seed examples to README.md

### PR DESCRIPTION
**Pull request by saada**
Added some example code for seeding the database. This is based on my answer here: http://stackoverflow.com/questions/28548137/one-to-many-relationships-with-factorymuffin-in-php/28612034#28612034

### CODE REVIEW

**Comment by scottrobertson** [comment]
I will leave this one to @GrahamCampbell, but my only suggestion is to use braces in the foreach loop :)

File: README.md [hunk 1]
```diff
@@ -159,7 +159,19 @@ $fm->define('Bar')->addDefinition('baz', Faker::date('Y-m-d'));
...
 
-The `create` function will create and save your model, and will also save anything you generate with the `Factory` generator too. If you want to create multiple instances, check out the seed `seed` function, which accepts an additional argument at the start which is the number of models to generate in the process. The `seed` function will effectively be calling the `create` function over and over. It should be noted that you can set a custom save function before you get going with the `setSaveMethod` function. Also, a reminder that the `instance` function is still available if you don't want database persistence.
+The `create` function will create and save your model, and will also save anything you generate with the `Factory` generator too. If you want to create multiple instances, check out the seed `seed` function, which accepts an additional argument at the start which is the number of models to generate in the process. The `seed` function will effectively be calling the `create` function over and over. 
+
+For example, let's create a `User` model and associate multiple `Location` and `Email` models to each one. Each `Email` will also have multiple `Token` models.
+```php
```
Comment (GrahamCampbell) [inline README.md]: Leave a new line before this code block please.

**Comment by saada** [comment]
there... anything else :smile_cat: ?

**Comment by GrahamCampbell** [comment]
Sorry this is taking so long. I'm going to give the documentation some love in the first week of July this year.

**Comment by GrahamCampbell** [comment]
Thanks.

**Pull request closed without merging**